### ISSUE TITLE
Single hazard risk module not working

### ISSUE THREAD

**Issue by shakyaanice** [ashokdahal/RiskChanges#53]
```
Traceback (most recent call last):
  File "/home/sdss/.virtualenvs/sdss/lib/python3.8/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/sdss/.virtualenvs/sdss/lib/python3.8/site-packages/django/core/handlers/base.py", line 115, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/sdss/.virtualenvs/sdss/lib/python3.8/site-packages/django/core/handlers/base.py", line 113, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/sdss/.virtualenvs/sdss/lib/python3.8/site-packages/django/views/decorators/csrf.py", line 54, in wrapped_view
    return view_func(*args, **kwargs)
  File "/home/sdss/.virtualenvs/sdss/lib/python3.8/site-packages/rest_framework/viewsets.py", line 114, in view
    return self.dispatch(request, *args, **kwargs)
  File "/home/sdss/.virtualenvs/sdss/lib/python3.8/site-packages/rest_framework/views.py", line 505, in dispatch
    response = self.handle_exception(exc)
  File "/home/sdss/.virtualenvs/sdss/lib/python3.8/site-packages/rest_framework/views.py", line 465, in handle_exception
    self.raise_uncaught_exception(exc)
  File "/home/sdss/.virtualenvs/sdss/lib/python3.8/site-packages/rest_framework/views.py", line 476, in raise_uncaught_exception
    raise exc
  File "/home/sdss/.virtualenvs/sdss/lib/python3.8/site-packages/rest_framework/views.py", line 502, in dispatch
    response = handler(request, *args, **kwargs)
  File "/opt/sdss/django/sdss/risk/views.py", line 99, in create
    risk_calculation(risk_id)
  File "/opt/sdss/django/sdss/risk/tasks.py", line 49, in risk_calculation
    ComputeRisk(CONN_STR, loss_id, risk_id)
  File "/home/sdss/.virtualenvs/sdss/lib/python3.8/site-packages/RiskChanges/ComputeRisk.py", line 65, in ComputeRisk
    risk = calculateRisk(lossdf, columns, probs)
  File "/home/sdss/.virtualenvs/sdss/lib/python3.8/site-packages/RiskChanges/ComputeRisk.py", line 48, in calculateRisk
    aal = dutch_method(xx, yy)
  File "/home/sdss/.virtualenvs/sdss/lib/python3.8/site-packages/RiskChanges/ComputeRisk.py", line 10, in dutch_method
    AAL = auc(yy, xx)+(xx[0]*yy[0])
  File "/home/sdss/.virtualenvs/sdss/lib/python3.8/site-packages/sklearn/metrics/_ranking.py", line 97, in auc
    raise ValueError("x is neither increasing nor decreasing "

Exception Type: ValueError at /api/risk/
Exception Value: x is neither increasing nor decreasing : [0.02 0.05 0.01].
```

**Comment by ashokdahal** [comment]
solved by previous comment, basically sorted the return period and the loss values to create a perfect graph where x is in increasing order. @shakyaanice @iamtekson  please check if it works.

**Comment by iamtekson** [comment]
Updated the latest code. @shakyaanice, can you test it now?

**Comment by iamtekson** [comment]
Solved by #89
